src/inspector/view/qml/MuseScore/Inspector/notation/lines/internal/LineTextSettingsTab.qml

```diff
@@ -67,13 +67,34 @@ FocusableItem {
 
         SeparatorLine { anchors.margins: -12 }
 
+        SpinBoxPropertyView {
+            id: beginLineGapFromTextControl
```

Comment:
I'd adjust this ID to `gapBetweenTextAndLineControl` to match the name of the control.

Reply:
Changed!